### PR TITLE
Add .NET MAUI Single Project Sample

### PR DESCRIPTION
**Pull request by Redth**
Add .NET MAUI Single Project Sample

### CODE REVIEW

File: HelloMaui/HelloMaui.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
```
Comment (jonathanpeppers) [inline HelloMaui/HelloMaui.csproj:3]: Should we change this, so it still builds on Windows:
```suggestion
    <TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('osx'))">net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
    <TargetFrameworks Condition="!$([MSBuild]::IsOSPlatform('osx'))">net6.0-android;net6.0-ios</TargetFrameworks>
```

File: README.md [hunk 2]
```diff
@@ -76,18 +76,12 @@ To launch the iOS project on a simulator:
...
-To launch the Forms project, you will need to specify a `$(TargetFramework)` via the `-f` switch:
-
-    dotnet build HelloForms -t:Run -f net6.0-android
-    dotnet build HelloForms -t:Run -f net6.0-ios
-
-> NOTE: You may need to add the `--no-restore` switch until
-> [dotnet#15485](https://github.com/dotnet/sdk/issues/15485) is
-> resolved.
+    dotnet build HelloMaui -t:Run -f net6.0-android
+    dotnet build HelloMaui -t:Run -f net6.0-ios
```
Comment (jonathanpeppers) [inline README.md:84]: Should we put the `--no-restore` switch in the example? A lot of people run into that bug.

Comment (mattleibow) [reply README.md:84]: I see we removed the note

> NOTE: You may need to add the `--no-restore` switch until
> [dotnet#15485](https://github.com/dotnet/sdk/issues/15485) is
> resolved.

Also, we need catalyst!

File: HelloMaui/HelloMaui.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0-android;net6.0-ios</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">$(TargetFrameworks);net6.0-maccatalyst</TargetFrameworks>
+    <OutputType>Library</OutputType>
```
Comment (mattleibow) [inline HelloMaui/HelloMaui.csproj:5]:
```suggestion
    <OutputType>Exe</OutputType>
```

File: README.md (hunk 2 above)
Comment (mattleibow) [inline README.md:84]:
```suggestion
> NOTE: You may need to add the `--no-restore` switch until
> [dotnet#15485](https://github.com/dotnet/sdk/issues/15485) is
> resolved.

    dotnet build HelloMaui -t:Run -f net6.0-android
    dotnet build HelloMaui -t:Run -f net6.0-ios
    dotnet build HelloMaui -t:Run -f net6.0-maccatalyst
```

**Review by savariya28 — COMMENTED** [review]
#

**Pull request merged**